{"url":"https://api.github.com/repos/owncloud/core/issues/6360","labels_url":"https://api.github.com/repos/owncloud/core/issues/6360/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/6360/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/6360/events","html_url":"https://github.com/owncloud/core/issues/6360","id":24192017,"number":6360,"title":"[oc6 & mirall 1.4.2 & 1.5rc1] mirall fails to sync to attached webdav due to \"server does not prvide etag\"","user":{"login":"niederfuchs","id":938221,"avatar_url":"https://gravatar.com/avatar/e05edad65c661448f9a0556e82a90651?d=https%3a%2f%2fidenticons.github.com%2f40c8691629471ff97163dccb3ebc5cf9.png&r=x","gravatar_id":"e05edad65c661448f9a0556e82a90651","url":"https://api.github.com/users/niederfuchs","html_url":"https://github.com/niederfuchs","followers_url":"https://api.github.com/users/niederfuchs/followers","following_url":"https://api.github.com/users/niederfuchs/following{/other_user}","gists_url":"https://api.github.com/users/niederfuchs/gists{/gist_id}","starred_url":"https://api.github.com/users/niederfuchs/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/niederfuchs/subscriptions","organizations_url":"https://api.github.com/users/niederfuchs/orgs","repos_url":"https://api.github.com/users/niederfuchs/repos","events_url":"https://api.github.com/users/niederfuchs/events{/privacy}","received_events_url":"https://api.github.com/users/niederfuchs/received_events","type":"user","site_admin":false},"labels":[],"state":"closed","assignee":null,"milestone":null,"comments":1,"created_at":"2013-12-12t16:57:05z","updated_at":"2013-12-12t17:00:28z","closed_at":"2013-12-12t17:00:28z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"it is hard to determine whether this error belongs to core or to mirall. if you connect two owncloud 6 instances (a and b) directly via webdav - inserting one folder from a into instance b. then connect mirall 1.4.2 or 1.5 rc1 with instance b - it can download the files from the folder from instance a. however uploading files to instance via the mirall connected to instance b - does not work. the client produces a message stating that the server did not provide an etag\"- the error log of instance b shows the following entries:\r\n\r\n```\r\nerror\twebdav client\tresource not found.\t2013-12-12t16:38:01+00:00\r\nerror\twebdav\t\\oc\\files\\filesystem::rename() failed\t2013-12-12t16:34:02+00:00\r\n```","closed_by":{"login":"pvince81","id":277525,"avatar_url":"https://gravatar.com/avatar/e446b529679416eb70eafb4ca4da5bbe?d=https%3a%2f%2fidenticons.github.com%2f89d7c394e6dcdf87a91c75ac3b2ccbe8.png&r=x","gravatar_id":"e446b529679416eb70eafb4ca4da5bbe","url":"https://api.github.com/users/pvince81","html_url":"https://github.com/pvince81","followers_url":"https://api.github.com/users/pvince81/followers","following_url":"https://api.github.com/users/pvince81/following{/other_user}","gists_url":"https://api.github.com/users/pvince81/gists{/gist_id}","starred_url":"https://api.github.com/users/pvince81/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/pvince81/subscriptions","organizations_url":"https://api.github.com/users/pvince81/orgs","repos_url":"https://api.github.com/users/pvince81/repos","events_url":"https://api.github.com/users/pvince81/events{/privacy}","received_events_url":"https://api.github.com/users/pvince81/received_events","type":"user","site_admin":false}}